### PR TITLE
(SIMP-1695) Update global catalysts

### PR DESCRIPTION
**Pull request by nick-markowski**
SIMP-2157 #close

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -195,15 +202,16 @@
     Package['nfs-utils'] -> Package['autofs']
     Service['autofs'] ~> Service[$::nfs::service_names::rpcbind]
 
-    if $::nfs::use_stunnel {
+    if $stunnel {
```
Comment (trevor-vaughan) [inline manifests/init.pp]: This needs to be reverted.

If you're using NFS, the use of Stunnel here is *only* dictated by the use of stunnel in NFS.

**Comment by nick-markowski** [comment]
This needs more work than requested, don't simply fix then merge.

**Comment by nick-markowski** [comment]
All dependencies merged (finally!) Working on a big patch set for this

**Comment by trevor-vaughan** [comment]
I've updated everything and validated that it works via the spec tests and the (pending) NFS acceptance tests.

**Pull request merged**